pkg/proxy/healthcheck/proxier_health.go

```diff
@@ -147,14 +148,6 @@ func (h healthzHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 		resp.WriteHeader(http.StatusServiceUnavailable)
 	} else {
 		resp.WriteHeader(http.StatusOK)
-
-		// In older releases, the returned "lastUpdated" time indicated the last
-		// time the proxier sync loop ran, even if nothing had changed. To
-		// preserve compatibility, we use the same semantics: the returned
-		// lastUpdated value is "recent" if the server is healthy. The kube-proxy
-		// metrics provide more detailed information.
-		lastUpdated = currentTime
```

Comment:
This needs to be kept

Reply:
Do we also kept the `lastUpdated = currentTime` ?  I don't get it 😅,  When `kube-proxy` is healthy，Users will see the time of `lastUpdated` and the time of `currentTime `(`currentTime` > `lastUpdated`), Is it better? 

If `lastUpdated = currentTime`, User may feels a little confused..

Reply:
Read the comment. It's a backward-compatibility issue. Some clients may assume that kube-proxy is unhealthy if `lastUpdated` is more than 30 seconds in the past. The data is effectively meaningless at this point, but we don't want to change it.

Reply:
Thanks!